### DISCUSSION TITLE
我添加的账号是本地储存，上传大文件100%后一直显示后端处理中。

### DISCUSSION THREAD

**Discussion by 6396036** [AlistGo/alist#435] [Q&A]
我添加的账号是本地储存，上传大文件（200+M），上传大文件100%后一直显示后端处理中，等了半小时也没变化。
刷新网页也没成功。

服务器php5.6，已修改
post_max_size = 10240M

**Comment by 6396036** [comment]
补充一下问题：
经测试，用VPS 的IP+5244 上传可以
但是经过域名反代后就不行了，可能是反代出的问题，大佬有没有反代配置文件。

**Comment by xhofe** [comment]
https://www.bing.com/search?q=nginx+client_max_body_size

**Reply by 6396036** [reply]
改过了       用域名传还是不行

**Reply by xhofe** [reply]
那自行搜索吧 非程序问题了 nginx我也不清楚
